frontend/javascripts/oxalis/view/left-border-tabs/modals/add_volume_layer_modal.tsx

```diff
@@ -17,11 +17,67 @@ import {
 import {
   getAllReadableLayerNames,
   getVolumeTracingLayers,
-  validateReadableLayerName,
 } from "oxalis/model/accessors/volumetracing_accessor";
+import messages from "messages";
 import InputComponent from "oxalis/view/components/input_component";
 import api from "oxalis/api/internal_api";
 import Toast from "libs/toast";
+
+export type ValidationResult = { isValid: boolean; message: string };
+export function checkForLayerNameDuplication(
+  readableLayerName: string,
+  allReadableLayerNames: string[],
+): ValidationResult {
+  const layerNameDoesNotExist = _.countBy(allReadableLayerNames)[readableLayerName] <= 0;
```

Comment:
For layer names that do not exist, `_.countBy(allReadableLayerNames)[readableLayerName]` will be `undefined`, resulting in `undefined <= 0` which is `false`. So I think in its current state this method will never return that the layer name is valid.

Something like
```suggestion
  const layerNameDoesNotExist = !allReadableLayerNames.includes(readableLayerName);
```

should do the job.